phpBB/adm/style/acp_attachments.html

```diff
@@ -110,7 +110,7 @@ <h2>{L_UPLOADING_FILES}</h2>
 <!-- ELSEIF S_EXTENSION_GROUPS -->
 
 	<!-- IF S_EDIT_GROUP -->
-		<script type="text/javascript" defer="defer">
+		<script defer="defer">
```

Comment:
As per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer
Having the `defer="defer"` for an in-line script without a `src=""` is pointless.